src/components/Form.tsx

```diff
@@ -1,98 +1,16 @@
 import React from 'react';
-import Button from '@material-ui/core/Button';
-import TextField from '@material-ui/core/TextField';
-import Grid from '@material-ui/core/Grid';
-import Box from '@material-ui/core/Box';
-import Typography from '@material-ui/core/Typography';
-import { makeStyles } from '@material-ui/core/styles';
-import Container from '@material-ui/core/Container';
+import NamePicker from './NamePicker';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import CategoryPicker from './CategoryPicker';
 
 export default function SearchForm(): JSX.Element {
```

Comment:
Since the component is now named SearchForm, after submitting this CL you might want to do another quick one that renames the file from `Form.tsx` to `SearchForm.tsx`.

Reply:
Noted, will do!